src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,15 @@
 // Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const {
+    email,
+    name,
+  } = user;
```

Comment:
Be more consistent with line length

it is ok to have single line here

```suggestion
  const { email, name } = user;
```